packages/navigation/src/index.js

```diff
@@ -86,16 +86,18 @@ export const getScreenFromPath = ( path = getPath() ) => {
 /**
  * Get an array of IDs from a comma-separated query parameter.
  *
- * @param {string} queryString string value extracted from URL.
- * @return {Array} List of IDs converted to numbers.
+ * @param {string} [queryString=''] string value extracted from URL.
+ * @return {Array<number>} List of IDs converted to an arry of unique integers.
```

Comment:
```suggestion
 * @return {Array<number>} List of IDs converted to an array of unique integers.
```

Also should the return type still be an array when it uses `Set`?

Reply:
It uses `Set` internally but still returns an array https://github.com/woocommerce/woocommerce-admin/pull/7028/files/690e5d73bd611674831e11f69f29e1a832c52341#diff-6efa564f97a1e5bbd69c59abf45e53f218b9984f77817a13cbdfd4b64de57518R93